notebooks/catvton/catvton.ipynb

```diff
@@ -0,0 +1,261 @@
+{
```

Comment:
<p>&gt;to save it as XML file.</p><p> </p><p> </p><p> actually, openvino IR is not just XML, XML contains only model graph and operations definitions while weights stored in bin file. Such comments may mislead newcomers (e.g. in my practice, I sow cases when people really thought that only XML is enough for inference and try to run model after remove bin file ), could you please rephrase avoiding mention xml</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2453/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>